plugins/mock/test/mockDatabase.test.ts

```diff
@@ -0,0 +1,114 @@
+import {describe, expect, test} from 'vitest';
+import {mockDatabase} from '../src';
+import {Bundle} from '@hot-updater/core';
+
+const DEFAULT_BUNDLES_MOCK: Bundle[] = [
+  {
+    id: '1',
+    enabled: true,
+    fileUrl: 'https://example.com/bundle.js',
+    shouldForceUpdate: false,
+    fileHash: '1234',
+    gitCommitHash: '5678',
+    platform: 'ios',
+    targetAppVersion: '',
+    message: null,
+  },
+  {
+    id: '2',
+    enabled: true,
+    fileUrl: 'https://example.com/bundle.js',
+    shouldForceUpdate: false,
+    fileHash: '1234',
+    gitCommitHash: '5678',
+    platform: 'ios',
+    targetAppVersion: '',
+    message: null,
+  },
+];
+
+describe('mockDatabase', () => {
+  test('should return a database plugin', async () => {
+    const plugin = mockDatabase({})({cwd: ''});
+
+    const bundles = await plugin.getBundles();
+
+    expect(bundles).toEqual([]);
+  });
+
+  test('should return a database plugin with initial bundles', async () => {
+    const plugin = mockDatabase({
+      initialBundles: DEFAULT_BUNDLES_MOCK,
+    })({cwd: ''});
+
+    const bundles = await plugin.getBundles();
+
+    expect(bundles).toEqual(DEFAULT_BUNDLES_MOCK);
+  });
+
+  test('should append a bundle', async () => {
+    const plugin = mockDatabase({})({cwd: ''});
+
+    await plugin.appendBundle(DEFAULT_BUNDLES_MOCK[0]);
+
+    const bundles = await plugin.getBundles();
+
+    expect(bundles).toEqual([DEFAULT_BUNDLES_MOCK[0]]);
```

Comment:
```suggestion
    const bundles = await plugin.getBundles();
    const bundles = await plugin.commitBundle();

    expect(bundles).toEqual([DEFAULT_BUNDLES_MOCK[0]]);
```



It seems that, like the review above, changes should not be made once confirmed before commitBundle.